test/thirdpartyemailpassword/emailverify.test.js

```diff
@@ -239,6 +268,68 @@ describe(`emailverify: ${printPath("[test/thirdpartyemailpassword/emailverify.te
         assert(emailToken !== null);
     });
 
+    it("test that providing your own email callback and make sure it is called (thirdparty user)", async function () {
```

Comment:
We need to add more tests like:
- Emailpassword recipe:
   - creating email password user and doing reset password and email verification email
   - Trying to send reset password to non existent user should not call your callback
- Third party recipe:
   - creating third party user and sending email verification to them.
- Third party email password recipe:
   - creating email password user and doing reset password and email verification email
   - creating third party user and doing email verification email
   - creating third party user and doing password reset -> should not call the callback
   - Trying to send reset password to non existent user should not call your callback.
- Passwordless recipe:
   - Sending email and SMS for login should call callback
-  Third party passwordless recipe:
   - Sending email and SMS for passwordless login should call callback
   - Email verification email for third party user -> should call callback
   - Email verification for passwordless user -> should not call callback